Library/Homebrew/keg.rb

```diff
@@ -467,13 +466,13 @@ def oldname_opt_record
     end
   end
 
-  # TODO: refactor to use keyword arguments.
-  def link(**options)
+  def link(verbose: false, dry_run: false, overwrite: false)
     raise AlreadyLinkedError, self if linked_keg_record.directory?
 
     ObserverPathnameExtension.reset_counts!
+    options = { verbose: verbose, dry_run: dry_run, overwrite: overwrite }
 
-    optlink(**options) unless options[:dry_run]
+    optlink(**options) unless dry_run
```

Comment:
These can just be passed as keyword arguments as well.

Reply:
In general, `options` should be refactored to keyword arguments throughout the whole call chain.